core/src/main/java/com/linecorp/armeria/common/AbstractHttpRequestBuilder.java

```diff
@@ -285,11 +274,10 @@ final HttpHeadersBuilder headersBuilder() {
 
     private RequestHeaders requestHeaders() {
         requestHeadersBuilder.path(buildPath());
-        if (cookies != null) {
-            requestHeadersBuilder.set(COOKIE, Cookie.toCookieHeader(cookies));
-        }
+        @SuppressWarnings("resource")
         final HttpData content = content();
         if (content == null || content.isEmpty()) {
+            //noinspection ReactiveStreamsUnusedPublisher
```

Comment:
cruft?

Reply:
No, it wasn't. I intentionally added it to suppress the warning. 
<img width="474" alt="image" src="https://user-images.githubusercontent.com/1866157/176187209-d6210324-db52-4ae7-b7f6-f4761bb611a5.png">